templates/etc/httpd/conf.d/_ssl_vhost.conf.erb

```diff
@@ -0,0 +1,7 @@
+# allow older yum clients to connect, see bz 647828
+SSLInsecureRenegotiation On
```

Comment:
:+1: for keeping this around, given older EL5/Centos5 clients need to connect in order to update